client-js/TUTORIAL.md

```diff
@@ -0,0 +1,238 @@
+# CEP-78 JavaScript Client Tutorial
+
+This tutorial outlines the usage of the JavaScript client available for the CEP-78 Enhanced NFT Standard.
+
+Further information on the CEP-78 Enhanced NFT Standard can be found [here](https://github.com/casper-ecosystem/cep-78-enhanced-nft).
+
+The client is available in the npm as [casper-cep78-js-client](https://www.npmjs.com/package/casper-cep78-js-client).
```

Comment:
The client is available in *npm* as [casper-cep78-js-client](https://www.npmjs.com/package/casper-cep78-js-client).